src/systemctl/systemctl.c

```diff
@@ -6504,6 +6504,12 @@ static int set_environment(int argc, char *argv[], void *userdata) {
         return 0;
 }
 
+static void invalid_callback(const char *p, void *userdata) {
+        _cleanup_free_ char *t = cescape(p);
+
+        log_debug("Ignoring invalid environment assignment \"%s\".", strnull(t));
```

Comment:
should this really just be debug?

Reply:
I used debug because on my system there's quite a few variables which we refuse to export:
Ignoring invalid environment assignment "LESS_TERMCAP_mb=\033[01;31m".
Ignoring invalid environment assignment "LESS_TERMCAP_md=\033[01;31m".
Ignoring invalid environment assignment "LESS_TERMCAP_me=\033[0m".
Ignoring invalid environment assignment "LESS_TERMCAP_se=\033[0m".
Ignoring invalid environment assignment "LESS_TERMCAP_so=\033[01;44;33m".
Ignoring invalid environment assignment "LESS_TERMCAP_ue=\033[0m".
Ignoring invalid environment assignment "LESS_TERMCAP_us=\033[01;32m".
Logging at non-debug level about all of them would be annoying. So we should either relax our rules to allow the rvalues used here, or keep logging at debug level.